src/utils/xhr-utils.js

```diff
@@ -222,8 +222,7 @@ export function getViewerInterceptResponse(win, ampdocSingle, input, init) {
       return viewer.isTrustedViewer();
     })
     .then(viewerTrusted => {
-      const isDevMode = getMode(win).development;
-      if (!viewerTrusted && !isDevMode) {
+      if (!viewerTrusted && !getMode(win).development) {
```

Comment:
Hmm, should this also be `getMode(win).localDev`?

Reply:
Looks like it was added in #14837, but `getMode().development` is set when `#development=1` is in the URL. We probably shouldn't allow this to be opted out via URL.

Reply:
right, I believe Su made the change and maybe there was some confusion as to what it meant. As there is localDev and then development. the latter pertains to the validation errors that are visible for development and the former is the actual v0 binary built for local testing. So yes, he probably meant localDev. I'll leave it as it was and can make the change separately. Thanks